source/PlayerInfo.cpp

```diff
@@ -52,6 +52,9 @@ this program. If not, see <https://www.gnu.org/licenses/>.
 #include <sstream>
 #include <stdexcept>
 
+#define KNOWN_OUTFIT_KEY "known outfit"
+#define VISITED_OUTFITTERS_KEY "visited outfitter"
```

Comment:
I've used it because it is used in more than one location, so if it changed for some reason, only one place needs change.
reduces the ability for errors

Reply:
may as well just define it in the namespace then, as we often do

Reply:
will look into it